suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -444,3 +454,63 @@ export const createImportedDeviceThunk = createThunk(
         }
     },
 );
+
+export const confirmAddressOnDeviceThunk = createThunk(
```

Comment:
Why do we need new method? Isn't this already solved somehow in desktop?

Reply:
I've extracted the desktop logic from `packages/suite` and moved it into `suite-common`. This thunk is basically refactor of the desktop method.